src/udns.mli

```diff
@@ -657,44 +557,35 @@ module Rr_map : sig
   module Tlsa_set : Set.S with type elt = Tlsa.t
   module Sshfp_set : Set.S with type elt = Sshfp.t
 
-  type _ k =
-    | Soa : Soa.t k
-    | Ns : (int32 * Domain_name.Set.t) k
-    | Mx : (int32 * Mx_set.t) k
-    | Cname : (int32 * Domain_name.t) k
-    | A : (int32 * Ipv4_set.t) k
-    | Aaaa : (int32 * Ipv6_set.t) k
-    | Ptr : (int32 * Domain_name.t) k
-    | Srv : (int32 * Srv_set.t) k
-    | Dnskey : (int32 * Dnskey_set.t) k
-    | Caa : (int32 * Caa_set.t) k
-    | Tlsa : (int32 * Tlsa_set.t) k
-    | Sshfp : (int32 * Sshfp_set.t) k
-    | Txt : (int32 * Txt_set.t) k
-
-  val equal_k : 'a k -> 'a -> 'b k -> 'b -> bool
+  type _ rr =
+    | Soa : Soa.t rr
+    | Ns : (int32 * Domain_name.Set.t) rr
```

Comment:
this does look better! is it worth having a type alias for the `int32` here to document what it represents in the RRs?